src/components/button/Button.vue

```diff
@@ -11,21 +11,26 @@
         @click="animateHandler($event)"
         @keyup.tab="tabbedIn"
     >
-        <VsIcon
-            v-if="icon"
-            class="align-self-center"
-            :name="icon"
-            :class="iconClasses"
-            size="xs"
-            :padding="0"
-        />
-        <!-- @slot The button content goes here -->
-        <span
-            class="vs-button__text"
-            :class="{ 'visually-hidden': iconOnly }"
+        <div
```

Comment:
Placing a div inside a button is not semantically correct in HTML. I reckon the best fix here would be to follow Bootstraps change and refactor our block buttons to use utilities